### PR TITLE
Add TripPy + CoCoAug (Li and Yavuz et al. 2020) in DST on MultiWOZ2.1

### PR DESCRIPTION
**Pull request by LeeShiyang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +97,7 @@ The belief state have three sections: semi, book and booked. Semi refers to slot
 <h3>End-to-End Modelling</h3>
 <div class="datagrid" style="width:500px;">
 <table>
-<thead><tr><th>(INFORM	+ SUCCESS)*0.5 +	BLEU</th><th colspan="3">MultiWOZ 2.0</th><th colspan="3">MultiWOZ 2.1</th></tr></thead>
+<thead><tr><th>(INFORM  + SUCCESS)*0.5 +  BLEU</th><th colspan="3">MultiWOZ 2.0</th><th colspan="3">MultiWOZ 2.1</th></tr></thead>
```
Comment (XiaoxueZang) [inline README.md:100]: Please revert unnecessary changes.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Multi-Domain Wizard-of-Oz dataset (MultiWOZ), a fully-labeled collection of human-human written conversations spanning over multiple domains and topics. At a size of 10k dialogues, it is at least one order of magnitude larger than all previous annotated task-oriented corpora.
 
 
-The newest, corrected version of the dataset is available at [MultiWOZ_2.2](https://github.com/budzianowski/multiwoz/blob/master/data/MultiWOZ_2.2) thanks to [the Google crew](https://arxiv.org/abs/2007.12720).
+The newest, corrected version of the dataset is available at [MultiWOZ_2.2](https://github.com/budzianowski/multiwoz/blob/master/data/MultiWOZ_2.2.zip) thanks to [the Google crew](https://arxiv.org/abs/2007.12720).
```
Comment (XiaoxueZang) [inline README.md:5]: Please revert this.

File: README.md (hunk 1 above)
Comment (LeeShiyang) [reply README.md]: I have reverted these unnecessary changes in the new commit "Revert unnecessary changes in readme."

File: README.md (hunk 2 above)
Comment (LeeShiyang) [reply README.md]: I have reverted this change in the new commit "Revert unnecessary changes in readme." If you have further questions, please let me know. Thank you.

**Comment by XiaoxueZang** [comment]
Thanks for the change. I didn't read your paper in details but I couldn't find 60.53 in your paper by Ctrl+F. Just want to make sure that this number is correct.

**Comment by LeeShiyang** [comment]
> Thanks for the change. I didn't read your paper in details but I couldn't find 60.53 in your paper by Ctrl+F. Just want to make sure that this number is correct.

Yes, it's correct but we just updated our paper about 12 hours ago in arxiv. I think the new version will appear soon. If you need the appearance of the new version, I can give you a quick update then. Thank you!

**Comment by LeeShiyang** [comment]
> Thanks for the change. I didn't read your paper in details but I couldn't find 60.53 in your paper by Ctrl+F. Just want to make sure that this number is correct.

Hi Xiaoxue, our new version is available in arxiv. You should be able to find 60.53 in our paper by Ctrl+F. Could you please approve our pull request if you think there is no problem after your examination? Thank you.

File: README.md [hunk 3]
```diff
@@ -51,6 +51,7 @@ The belief state have three sections: semi, book and booked. Semi refers to slot
 <tr><td><a href="https://arxiv.org/abs/2005.02877">TripPy</a> (Heck et al. 2020)</td><td></td><td></td><td>55.3</td><td></td><td></td><td></td></tr>
 <tr><td><a href="https://arxiv.org/pdf/2005.00796.pdf">SimpleTOD</a> (Hosseini-Asl et al. 2020)</td><td></td><td></td><td>56.45</td><td></td><td></td><td></td></tr>
 <tr><td><a href="https://arxiv.org/pdf/2009.13570.pdf">ConvBERT-DG + Multi</a> (Mehri et al. 2020)</td><td></td><td></td><td>58.7</td><td></td><td></td><td></td></tr>
+<tr><td><a href="https://arxiv.org/pdf/2010.12850.pdf">TripPy + CoCoAug</a> (Li and  Yavuz et al. 2020)</td><td></td><td></td><td>60.53</td><td></td><td></td><td></td></tr>
```
Comment (HuangLK) [inline README.md:54]: Trippy exists [data leakage problem](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L446), it is not a solid work.

Comment (LeeShiyang) [reply README.md:54]: > Trippy exists [data leakage problem](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L446), it is not a solid work.

The line you refer to is for system act label, right? I think some other SOTA models also utilize additional annotation information (e.g. NADST (Le et al., 2019) uses system action and DSTQA (Zhou & Small, 2019) uses knowledge graph). Is this a problem? Thank you.

Comment (XiaoxueZang) [reply README.md:54]: Hi HuangLK, thanks for raising the issue. Please discuss with Shiyang and TripPy's authors about it and we'd like to be updated on the final conclusion! If there indeed exists the problem, please send a pull request to correct the number or add an annotation that these numbers are overestimated.

Comment (LeeShiyang) [reply README.md:54]: > Hi HuangLK, thanks for raising the issue. Please discuss with Shiyang and TripPy's authors about it and we'd like to be updated on the final conclusion! If there indeed exists the problem, please send a pull request to correct the number or add an annotation that these numbers are overestimated.

Hi Xiaoxue, 

The link HuangLK refers to is the additional annotation from multiwoz dataset itself rather than test set leakage. See Table 1 of SimpleTOP in NeurIPS 2020 (https://arxiv.org/pdf/2005.00796.pdf). Current SOTA DST model ConvBERT-DG + Multi in the leaderboard also utilize additional open-dialogue and Dialogue GLUE dataset. Because there doesn't exist test set leakage, I think these methods are all reasonable models. Does my reply answer your questions? Thank you.

Comment (HuangLK) [reply README.md:54]: The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the  value from dialog action is not equal to ground-truth value.

Comment (jianguoz) [reply README.md:54]: > The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.

@HuangLK @LeeShiyang Hi, have you come to an agreement regarding `Trippy exists data leakage problem, it is not a solid work.`

Comment (LeeShiyang) [reply README.md:54]: > The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.

I think such a process is like a semantic dictionary for mapping predictions to ground-truth, which is widely used in various DST works. Actually, different works have different pre-processing/post-processing/cleaning methods, which may take effect in the final performance.

Comment (LeeShiyang) [reply README.md:54]: > > The line I refer to means that the final informed values(from dialog actions) are checked with ground-truth values. See [the code](https://gitlab.cs.uni-duesseldorf.de/general/dsml/trippy-public/-/blob/master/dataset_multiwoz21.py#L267),the `informed_value` will be set to `none` if the value from dialog action is not equal to ground-truth value.
> 
> @HuangLK @LeeShiyang Hi, have you come to an agreement regarding `Trippy exists data leakage problem, it is not a solid work.`

Hi Jianguo, right now I understand what HuangLK means. Thank you.
